File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Lab/Solutions/P07_WorkingHours.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

	int time;
	string day;
	cin >> time >> day;

	string output;
	if (day == "Sunday" || time < 10 || 18 < time) {
		output = "closed";
	} else {
		output = "open";
	}

	cout << output << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Lab/Solutions/P07_WorkingHours.py
hour = int(input())
day = input()

if 0 <= hour <= 24:
    if day == 'Monday':
        if 10 <= hour <= 18:
            print('open')
        else:
            print('closed')
    elif day == 'Tuesday':
        if 10 <= hour <= 18:
            print('open')
        else:
            print('closed')
    elif day == 'Wednesday':
        if 10 <= hour <= 18:
            print('open')
        else:
            print('closed')
    elif day == 'Thursday':
        if 10 <= hour <= 18:
            print('open')
        else:
            print('closed')
    elif day == 'Friday':
        if 10 <= hour <= 18:
            print('open')
        else:
            print('closed')
    elif day == 'Saturday':
        if 10 <= hour <= 18:
            print('open')
        else:
            print('closed')
    elif day == 'Sunday':
        print('closed')
